package.json

```diff
@@ -54,6 +54,7 @@
     "chalk": "^1.0.0",
     "dot-prop": "^2.2.0",
     "get-stdin": "^5.0.1",
+    "lodash": "^4.11.1",
```

Comment:
Do we really want to pull in lodash for it being used once? Seems like a lot of overhead for a single function call. There is already `array-without` which would be much more lightweight.